components/02-elements/form/select/select.js

```diff
@@ -13,8 +13,9 @@
         list: 'select__field-list',
         input: 'select__field-input',
         item: 'select__field-item',
-        listSingle: 'select__field-list--single'
-      }
+        listSingle: 'select__field-list--single',
+      },
+      itemSelectText: ''
```

Comment:
What's the point of this change? You are not using it anywhere.

Reply:
Because by default there is value 'Press to choose' or sth like that and in shorter selects it is cut.